docs/build/custom.md

```diff
@@ -113,80 +113,122 @@ The `MinSizeRel` configuration will produce the smallest binary size.
+
+3. Use the local pods.
 
-Note: The onnxruntime-mobile-objc pod depends on the onnxruntime-mobile-c pod. If the released onnxruntime-mobile-objc pod is used, this dependency is automatically handled. However, if a local onnxruntime-mobile-objc pod is used, the local onnxruntime-mobile-c pod that it depends on also needs to be specified in the Podfile.
+    For example, update the Podfile to use the local onnxruntime-mobile-objc pod instead of the released one:
+
+    ```diff
+    -  pod 'onnxruntime-mobile-objc'
+    +  pod 'onnxruntime-mobile-objc', :path => "/path/to/staging/dir/onnxruntime-mobile-objc"
+    +  pod 'onnxruntime-mobile-c', :path => "/path/to/staging/dir/onnxruntime-mobile-c"
+    ```
+
+    Note: The onnxruntime-mobile-objc pod depends on the onnxruntime-mobile-c pod. If the released onnxruntime-mobile-objc pod is used, this dependency is automatically handled. However, if a local onnxruntime-mobile-objc pod is used, the local onnxruntime-mobile-c pod that it depends on also needs to be specified in the Podfile.
 
 ### Android
 
-_[This section is coming soon]_
+To produce an Android AAR package, use the [build_custom_android_package.py](https://github.com/microsoft/onnxruntime/blob/master/tools/android_custom_build/build_custom_android_package.py) script from the ONNX Runtime repo.
 
-### Python
+The script can be used from within the repo or outside of it. Copy its [containing directory](https://github.com/microsoft/onnxruntime/blob/master/tools/android_custom_build) for usage outside of the repo.
 
-If you wish to use the ONNX Runtime python bindings with a minimal build, exceptions must be enabled due to Python requiring them.
+1. Run the build script.
 
-Remove `--disable_exceptions` and add `--build_wheel` to the build command in order to build a Python Wheel with the ONNX Runtime bindings.
+    For example:
 
-A .whl file will be produced in the build output directory under the `<config>/dist` folder.
+    ```bash
+    python3 tools/android_custom_build/build_custom_android_package.py \
+      --onnxruntime_branch_or_tag v1.11.0 \
+      --include_ops_by_config /path/to/ops.config \
+      --build_settings /path/to/build_settings.json \
+      /path/to/working/dir
+    ```
 
-* The Python Wheel for a Windows MinSizeRel build using build.bat would be in `<ONNX Runtime repository root>\build\Windows\MinSizeRel\MinSizeRel\dist\`
-* The Python Wheel for a Linux MinSizeRel build using build.sh would be in `<ONNX Runtime repository root>/build/Linux/MinSizeRel/dist/`
+    This will do a custom build and create the Android AAR package for it in /path/to/working/dir.
 
-The wheel can be installed using `pip`. Adjust the following command for your platform and the whl filename.
+    Specify the ONNX Runtime version you want to use with the `--onnxruntime_branch_or_tag` option. The script uses a separate copy of the ONNX Runtime repo in a Docker container so this is independent from the containing ONNX Runtime repo's version.
 
```

Comment:
How many steps are involved if I wanted to do a build and create the AAR with an existing repo? Wondering if it would be worth documenting that too, although that's more likely useful to an ORT developer than an external user.

Reply:
we could update the Dockerfile to (optionally) use an existing repo. it is not currently set up to do that though.
pushing a development branch and specifying that branch on the command line would also work.